core/services/chainlink/config_general.go

```diff
@@ -32,6 +32,9 @@ type generalConfig struct {
 	effectiveTOML string // with default values included
 	c             *Config
 
+	secretsInputTOML string
```

Comment:
I think we can probably just drop this, no?
We don't want to log it - is there any other reason to hang on to it?